docs/spec/src/integration/spec/6-secure-integration.md

```diff
@@ -149,6 +148,10 @@ In the diagram, the top row shows L1 blocks every 12 s; the smaller squares are
 
 However, if the RecencyWindowSize is configured to be 0, the entire recency check is skipped. It is strongly not recommended to set it to 0, as it allows a malicious or misbehaving batcher to submit an AltDACommitment whose blob has been pruned by the DA network. An altda commitments is considered valid and can be processed by the next stage of the eigenda blob derivation.
 
+#### Protocol controlled recency window size
+
+The RecencyWindowSize is determined by `offchainDerivationVersion` if the integration uses `EigenDACertV4`. For `offchainDerivationVersion=0`, the RecencyWindowSize is 14400 measured in number of L1 blocks, which corresponds to 48 hours. `EigenDACertV2` and `EigenDACertV3` do not contain `offchainDerivationVersion` in the DACert, the corresponding RecencyWindowSize is `0`, effectively disabling the recency check.
```

Comment:
nit:
```suggestion
The RecencyWindowSize is determined by `offchainDerivationVersion` if the integration uses a DA Cert with version >= 4. For `offchainDerivationVersion=0`, the RecencyWindowSize is 14400 measured in number of Ethereum blocks (assuming 12 second block production time), roughly corresponding to 48 hours. Any DA Cert before V4 isn't checked for recency (i.e,`RecencyWindowSize=0`) since there's no `offchainDerivationVersion` field present in the legacy DA Certs. 
```